actions.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+set-password:
+  description: Change the admin user's password, which is used by charm. It is for internal charm users and SHOULD NOT be used by applications.
+  params:
+    username:
+      type: string
+      description: The username, the default value 'root'. Possible values - 'root'.
```

Comment:
please excuse me, we should use user secrets for new charms instead of actions

Reply:
This has been reworked: The user actions `get-password` and `set-password` are no longer supported. Instead users can add a secret `admin-password=...`, grant it to the charm and add the secret-id as a configuration option `admin-password=secret_id`.